windows/src/desktop/setup/UfrmHTML.pas

```diff
@@ -33,17 +33,16 @@ interface
 
 uses
   Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
-  StdCtrls, ExtCtrls, OleCtrls, SHDocVw, EmbeddedWB,
-  SHDocVw_EWB, EwbCore, KeymanEmbeddedWB;
+  StdCtrls, ExtCtrls, OleCtrls, SHDocVw;
 
 type
   TfrmHTML = class(TForm)
     cmdOK: TButton;
     panHTML: TPanel;
-    web: TKeymanEmbeddedWB;
     cmdPrint: TButton;
     cmdBack: TButton;
     cmdForward: TButton;
+    web: TWebBrowser;
```

Comment:
Here we use an IE embedded browser because the Chromium components are not available in the installer. This is used to display just the license; we could move that to plain text.